bucket/eiskaltdcpp.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "2.4.2",
+    "description": "File sharing program using DC and ADC protocols",
+    "homepage": "https://github.com/eiskaltdcpp/eiskaltdcpp",
+    "license": "GPL-3",
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.sourceforge.net/project/eiskaltdcpp/files/Windows/EiskaltDC++-2.4.2_x86_64-portable.7z",
+            "hash": "sha1:9ed0cd684989e40a29d58e85960f908d780774c0"
+        },
+        "32bit": {
+            "url": "https://downloads.sourceforge.net/project/eiskaltdcpp/files/Windows/EiskaltDC++-2.4.2_x86-portable.7z",
+            "hash": "sha1:764df76901b038d460909f4422950082c70c6869"
+        }
+    },
+    "extract_dir": "EiskaltDC++-2.4.2_x86_64",
+    "bin": "eiskaltdcpp-daemon.exe",
+    "shortcuts": [
+        [
+            "EiskaltDC++.exe",
+            "EiskaltDC++"
+        ]
+    ],
+    "checkver": {
+        "url": "https://sourceforge.net/projects/eiskaltdcpp/rss?path=/Windows",
+        "regex": "portable"
```

```suggestion
        "regex": "Windows/EiskaltDC\\+\\+-([\\d.]+)_x86_64-portable\\.7z"
```
We usually match regex from **file names** because it is less likely to cause problems. For example, (1) some version only has Linux binaries, but not for Windows. (2) the website changes layout, making something like `<h2>` being modified.

For regex, [regex101.com](regex101.com) is a great tool to learn and verify your regex, with detailed help texts.

---

>We usually match regex from file names because it is less likely to cause problems. For example, (1) some version only has Linux binaries, but not for Windows. (2) the website changes layout, making something like <h2> being modified.

😢

---

>For regex, [regex101.com](https://regex101.com/) is a great tool to learn and verify your regex, with detailed help texts.

Awesome. BTW I check rss regex in https://grepfeed.sigwait.tk/?url=https%3A%2F%2Fsourceforge.net%2Fprojects%2Feiskaltdcpp%2Frss%3Fpath%3D%2FWindows&filter=Windows%2FEiskaltDC%5C%5C%2B%5C%5C%2B-%28%5B%5C%5Cd.%5D%2B%29_x86_64-portable%5C%5C.7z